Documentation/supermuc.rst

```diff
@@ -177,3 +181,267 @@ set compiler options, run cmake, and compile with:
+
+::
+
+  mkdir -p $HOME/Download
+  cd $HOME/Download
+
+4. Install UCX:
+
+:: 
+
+  $ git clone https://github.com/openucx/ucx.git
+  $ cd ucx
+  $ git checkout v1.10.x
+  $ ./autogen.sh
+  $ mkdir build && cd build
+
+  $ CC=$(which gcc) CXX=$(which g++) FC=$(which gfortran) \
+  ../contrib/configure-opt \
+  --prefix=$SEISSOL_INSTALL \
+  --build=powerpc64le-redhat-linux-gnu \
+  --host=powerpc64le-redhat-linux-gnu \
+  --disable-logging --disable-debug \
+  --disable-assertions --disable-params-check \
+  --disable-params-check --enable-optimizations \
+  --disable-assertions --disable-logging --with-pic \
+  --without-java \
+  --enable-mt \
+  --with-cuda=/cineca/prod/opt/compilers/cuda/11.0/none \
+  --with-gdrcopy \
+  --with-knem=/opt/knem-1.1.3.90mlnx1 \
+  --without-xpmem
+
+  $ make -j
+  $ make install
+  $ cd ../..
+
+5. Install OpenMPI:
+
+::
+
+  $ git clone --recursive -b v4.1.x https://github.com/open-mpi/ompi.git
+  $ cd ompi
+  $ ./autogen.pl
+  $ mkdir build && cd build
+
+  $ CC=$(which gcc) CXX=$(which g++) FC=$(which gfortran) \
+  CFLAGS="-I/opt/pmix/3.1.5/include" CPPFLAGS="-I/opt/pmix/3.1.5/include" \
+  ../configure \
+  --prefix=$SEISSOL_INSTALL \
+  --with-memory-manager=none \
+  --enable-static=yes \
+  --enable-shared \
+  --with-slurm \
+  --with-pmix=/opt/pmix/3.1.5 \
+  --with-ucx=$SEISSOL_INSTALL \
+  --with-libevent=/usr \
+  --with-hwloc=/usr \
+  --with-verbs \
+  --enable-mpirun-prefix-by-default \
+  --with-platform=/cineca/prod/build/compilers/openmpi/4.0.3/gnu--8.4.0/BA_WORK/openmpi-4.0.3/contrib/platform/mellanox/optimized \
+  --with-hcoll=/opt/mellanox/hcoll \
+  --with-cuda=/cineca/prod/opt/compilers/cuda/11.0/none \
+  --with-knem=/opt/knem-1.1.3.90mlnx1 \
+  --without-xpmem
+
+  $ make -j
+  $ make install
+  $ cd ../..
+
+6. Install HDF5:
+
+::
+
+  $ wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.5/src/hdf5-1.10.5.tar.gz
+  $ tar -xvf ./hdf5-1.10.5.tar.gz
+  $ cd hdf5-1.10.5
+  $ ./autogen.sh
+  $ mkdir build && cd build
+
+  $ CPPFLAGS="-fPIC ${CPPFLAGS}" CC=mpicc CXX=mpicxx FC=mpif90 \
+  ../configure \
+  --prefix=$SEISSOL_INSTALL \
+  --build=powerpc64le-redhat-linux-gnu \
+  --host=powerpc64le-redhat-linux-gnu \
+  --enable-parallel --with-zlib --disable-shared \
+  --enable-fortran
+
+  $ make -j
+  $ make install
+  $ cd ../..
+
+7. Installing netCDF:
+
+::
+
+  $ wget https://syncandshare.lrz.de/dl/fiJNAokgbe2vNU66Ru17DAjT/netcdf-4.6.1.tar.gz
+  $ tar -xvf ./netcdf-4.6.1.tar.gz
+  $ cd hdf5-1.10.5
+  $ ./autogen.sh
+
+  $ CFLAGS="-fPIC ${CFLAGS}" CC=h5pcc \
+  ./configure \
+  --prefix=$SEISSOL_INSTALL \
+  --build=powerpc64le-redhat-linux-gnu \
+  --host=powerpc64le-redhat-linux-gnu \
+  --enable-shared=no \
+  --disable-dap
+
+  $ make -j 
+  $ make install
+  $ cd ..
+
+8. Installing ParMetis:
+
+:: 
+
+  $ wget http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/parmetis-4.0.3.tar.gz
```

Comment:
it seems this block is not reckognized as a code block.

Reply:
Fixed. It was caused by trailing white spaces